### DISCUSSION TITLE
Can SteamKit run the game on any account?

### DISCUSSION THREAD

**Discussion by estan1slao** [SteamRE/SteamKit#1234] [General]
I have a question. Is it possible to launch a game on any of the accounts using SteamKit, so that I can play it?

Let's say I have logged in with multiple accounts on Steam and I want to play a specific game from the list of ALL games on all accounts. Can I launch the game from a specific account and start playing it without any issues?

**Comment by zxcvqwerasdf** [comment]
Games using steam_api.dll* to access steam functions, token retrieving, friends, messages, etc. You can retrive tokens by steamkit for game launching, but you can't send it to the game without game interception.
